yogstation/code/modules/surgery/shadowling/sling_organs.dm

```diff
@@ -0,0 +1,72 @@
+//Snowflake spot for putting sling organ related stuff
+/obj/item/organ/eyes/night_vision/alien/sling
+	name = "shadowling eyes"
+	desc = "The eyes of a spooky shadowling!"
+
+/obj/item/organ/internal/shadowtumor
+	name = "black tumor"
+	desc = "A tiny black mass with red tendrils trailing from it. It seems to shrivel in the light."
+	icon_state = "blacktumor"
+	w_class = 1
+	zone = "head"
+	slot = "brain_tumor"
+	var/health = 3
+
+/obj/item/organ/internal/shadowtumor/New()
+	..()
+	SSobj.processing |= src
+
+/obj/item/organ/internal/shadowtumor/Destroy()
+	SSobj.processing.Remove(src)
+	..()
+
+/obj/item/organ/internal/shadowtumor/process()
+	if(isturf(loc))
+		var/turf/T = loc
+		var/light_count = T.get_lumcount()
+		if(light_count > LIGHT_DAM_THRESHOLD && health > 0) //Die in the light
+			health--
+		else if(light_count < LIGHT_HEAL_THRESHOLD && health < 3) //Heal in the dark
+			health++
+		if(health <= 0)
+			visible_message("<span class='warning'>[src] collapses in on itself!</span>")
+			qdel(src)
+	else
+		health += 0.5
```

Comment:
Use something like ``/mob/living/proc/heal_overall_damage()`` to heal, don't directly edit health.